ir/src/lib.rs

```diff
@@ -81,11 +82,14 @@ impl AirIR {
             }
         }
 
+        let public_inputs = symbol_table.get_public_inputs();
+
```

Comment:
Since we also need the periodic columns, I think it would be better to replace the `into_periodic_columns` method with a single `into_declarations` method that consumes the `symbol_table` and returns a tuple of `(public_inputs, periodic_columns)`

Reply:
Thanks, fixed.